test/005-template-compile.test.js

```diff
@@ -66,4 +66,15 @@ describe('Template compile', function () {
             done();
         }).catch(done);
     });
+
+    it('theme with block partials and unknown helper', function (done) {
```

Comment:
I am curious why this case tests two things

1. usage of {{> @partial-block}}
2. invalid helper